packages/Checkbox/src/Checkbox.tsx

```diff
@@ -138,19 +156,26 @@ export default class Checkbox extends PureComponent {
   state = { checked: false };
 
   componentDidMount() {
+    if (this.inputRef.current == null) {
```

Comment:
This should just be for TS sake, right? I wonder if we should do `this.inputRef.current!.indeterminate`. If we start getting errors then we should switch away from `React.createRef` and use a function instead.